src/pages/Home.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import Chart from '../components/Chart';
+import Coin from '../components/Coin';
+import Header from '../components/Header';
+import { FetchCoinsFunc } from '../redux/CoinList/CoinList';
+import './Home.css';
+
+const Home = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const coins = useSelector((state) => state.CoinsReducer);
+  useEffect(() => {
+    dispatch(FetchCoinsFunc());
+  }, []);
+
+  const handleDetails = (id) => navigate(`/details/${id}`);
+  return (
+    <div data-testid="intro-page" className="data-cont">
+      <Header path="/" />
+      <Chart title1="Crypto" title2="Currency" count={`${coins.length} Coins`} />
+      <div>
+        <h3 className="coins-title">Coins By Name, Symbol, Price</h3>
```

Comment:
- [ ] **Required:** You are required to implement a function to filter the data you retrieved from the API and render the result to the page. 

![image](https://user-images.githubusercontent.com/27709832/175299689-fe4dd1a9-14c1-4c1e-b2fb-47a9aa3ad07f.png)